build_tools/cmake/run_tf_tests.sh

```diff
@@ -30,8 +31,18 @@ CMD=(
   --max-time 1800
 )
 
+TARGET_BACKEND=""
+
 if (( ${IREE_VULKAN_DISABLE} != 1 )); then
-  CMD+=(-D FEATURES=vulkan)
+  TARGET_BACKEND="vulkan"
+fi
+
+if (( ${IREE_VMVX_TESTS_ENABLE} == 1 )); then
+  TARGET_BACKEND="vmvx,${TARGET_BACKEND}"
+fi
+
+if [[ -n ${TARGET_BACKEND} ]]; then
+  CMD+=(-D FEATURES=${TARGET_BACKEND})
 fi
```

Comment:
Seems like these should be appending, not overwriting to `TARGET_BACKEND`?

Reply:
Oh, that's sort of happening already... @GMNGeoffrey is there some other bash pattern we usually use?

Reply:
Ah, maybe use an array like in https://github.com/openxla/iree/blob/main/build_tools/cmake/ctest_all.sh

`+=` to append to the array
`[*]` to expand all values when using it

Reply:
array expansion uses space as the separator by default unless IFS is overridden. an explicit string appending as in the current PR is simpler.

I can covert it to IFS + array if there's a strong preference :)

Reply:
Use array now. :)